app/report/residency_fr_taxes_fr/ReportResidencyFrTaxesFr.tsx

```diff
@@ -0,0 +1,435 @@
+    }),
+    [gainsAndLosses],
+  );
+
+  return (
+    <div className="container">
+      <div className="print:hidden">
+        <MessageBox level="warning" title="Disclaimer">
+          <p>
+            These calculations are for informational purposes only and should
+            not be considered financial advice.
+          </p>
+          <p>
+            Despite all the efforts that were put in creating this tool, it is
+            your responsibility to verify the results.
+          </p>
+          <p>
+            This{" "}
+            <a
+              href="/2021_mc-kenzie-taxes-presentation.pdf"
+              target="_blank"
+              className="underline"
+            >
+              guide
+            </a>{" "}
+            sent by equity team in 2021 was used to create this calculator
+          </p>
+        </MessageBox>
+        <div className="my-2">
+          Based on the <b>expanded</b> exports both for Gain And Losses (My
+          Account &gt; Gains and losses) and Benefit History (My Account &gt;
+          Benefit History) from Etrade.
+        </div>
+      </div>
+      {gainsAndLosses.length === 0 ? (
+        <div className="flex items-baseline justify-center gap-3">
+          <span>Import the Gains and Losses CSV file: </span>
+          <EtradeGainAndLossesFileInput setData={setGainsAndLosses} />
+        </div>
+      ) : isFetching ? (
+        <p>Loading...</p>
+      ) : (
+        <div className="container">
+          <div className="flex items-baseline justify-between gap-3 print:hidden">
+            <span>Gains and Losses</span>
+            <Button
+              label="Clear"
+              color="red"
+              onClick={() => setGainsAndLosses([])}
+            />
+          </div>
+          <div className="flex gap-2 items-baseline justify-items-start print:hidden">
+            <input
+              type="checkbox"
+              id="printMode"
+              checked={isPrintMode}
+              onChange={() => setIsPrintMode(!isPrintMode)}
+            />
+            <label htmlFor="printMode">Print mode</label>
+          </div>
+          <Section title="Summary">
+            <div className="px-6">
+              <dl className="grid grid-cols-2 ">
+                <dt className="font-bold">FR qualified SO</dt>
+                <dd>{counts.frQualifiedSo} events</dd>
+                <dt className="font-bold">FR qualified RSU</dt>
+                <dd>{counts.frQualifiedRsu} events</dd>
+                <dt className="font-bold">ESPP</dt>
+                <dd>{counts.espp} events</dd>
+                <dt className="font-bold">US qualified SO</dt>
+                <dd>{counts.usQualifiedSo} events</dd>
+                <dt className="font-bold">US qualified RSU</dt>
+                <dd>{counts.usQualifiedRsu} events</dd>
+              </dl>
+            </div>
+          </Section>
+          <Section title="Select Income Source and Anexes">
```

Comment:
I made a similar comment in another PR but the screenshot section could use a little love 😄